pkg/security/metrics/prometheus/metrics.go

```diff
@@ -0,0 +1,123 @@
+package prometheus
+
+import (
+	"sync"
+
+	"github.com/blang/semver"
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
I'd prefer we re-use k8s component-base like in https://github.com/openshift/library-go/blob/master/pkg/monitor/health/metrics.go since you're get a lot of setup for free from there. Also pls reach out to monitoring team to ensure these metrics are following their requirements.

Reply:
nice, I have switched to component- base logic and it looks much better now